### PR TITLE
sptribs_frontend has been implemented

### PR DESCRIPTION
**Pull request by geciciserkan**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/ST-1300

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: environments/demo/backend_lb_config.yaml [hunk 1]
```diff
@@ -396,6 +396,8 @@ gateways:
         component: case-api
       - product: sptribs
```
Comment (osheaf01) [inline environments/demo/backend_lb_config.yaml:397]: Can we remove the dss-backend? Or, if not yet, can we raise a followup ticket to do so when we can?

File: environments/ithc/backend_lb_config.yaml [hunk 2]
```diff
@@ -344,3 +344,5 @@ gateways:
         component: case-api
       - product: sptribs
         component: dss-backend
```
Comment (osheaf01) [inline environments/ithc/backend_lb_config.yaml:346]: As above, can we remove dss-backend?

File: environments/stg/backend_lb_config.yaml [hunk 3]
```diff
@@ -370,6 +370,8 @@ gateways:
         component: case-api
       - product: sptribs
```
Comment (osheaf01) [inline environments/stg/backend_lb_config.yaml:371]: As above, can we remove dss-backend?

**Review by osheaf01 — CHANGES_REQUESTED** [review]
Can we remove dss-backend?
If not, good to go.

File: environments/demo/backend_lb_config.yaml (hunk 1 above)
Comment (geciciserkan) [reply environments/demo/backend_lb_config.yaml:397]: we will do that by separate PR as we agreed

File: environments/stg/backend_lb_config.yaml (hunk 3 above)
Comment (geciciserkan) [reply environments/stg/backend_lb_config.yaml:371]: we will do that by separate PR as we agreed

File: environments/stg/backend_lb_config.yaml [hunk 4]
```diff
@@ -369,7 +369,7 @@ gateways:
...
         component: case-api
       - product: sptribs
-        component: dss-backend
+        component: frontend
```
Comment (thomast1906) [inline environments/stg/backend_lb_config.yaml:372]: the `frontend` to me is confusing as this is a `backend` appgw

**Review by thomast1906 — APPROVED** [review]
Questioned the reference to `frontend` with Serkan Gecici - they had confirmed:

```
that is the project name: sptribs-frontend
we can not change this project name
other project name was sptribs-dss-backend
we depreciated sptribs-dss-backend project
that is why we are removing as it is unused
I normally added sptribs-frontend
my friends said, please also remove unused project
so they are not same project
they are separate projects
```

**Review by osheaf01 — APPROVED** [review]
LGTM

**Pull request merged**